libs/database/src/lib/entities/site.entity.ts

```diff
@@ -122,6 +234,10 @@ export class Site extends Model<Site> {
   })
   seedsPlanted: Seeding[] | null;
 
+  get projectName() {
+    return this.project?.name;
+  }
```

Comment:
Please move this and `projectUuid` getter to both be just below the `project` declaration. It's useful to see at a glance what the dependencies on an association are.

Reply:
moved